tools/build_virtual_environment/dynamic_ve_base/contents/tmp/setup/create_demo_accounts.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+cd /tmp/setup
+
+config_file="/mnt/setup/atservers"
+
+while read -r line; do
+  echo "line: $line"
+  ./createConf $line # intentional word splitting
+done <"$config_file"
```

Comment:
instead of making several inline calls to ./createConf, we read the mounted config file and call createConf for each one.